core/Common.php

```diff
@@ -298,7 +298,7 @@ public static function safe_unserialize($string, $allowedClasses = [], $rethrow
     {
         try {
             // phpcs:ignore Generic.PHP.ForbiddenFunctions
-            return unserialize($string, ['allowed_classes' => empty($allowedClasses) ? false : $allowedClasses]);
+            return unserialize($string ?? '', ['allowed_classes' => empty($allowedClasses) ? false : $allowedClasses]);
```

Comment:
note to myself: it'll return always false which it also did before https://3v4l.org/qC9u3